modin/_version.py

```diff
@@ -96,9 +112,10 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
         if verbose:
             print("unable to find command, tried %s" % (commands,))
         return None, None
-    stdout = p.communicate()[0].strip()
+    stdout: str
+    stdout = p.communicate()[0].strip()  # type: ignore
     if sys.version_info[0] >= 3:
-        stdout = stdout.decode()
+        stdout = stdout.decode()  # type: ignore
```

Comment:
Why ignore?

Reply:
`stdout` obtained from `Popen.communicate` is a bytes object in Python 3+ (and I think a string in Python 2). The type of this variable is going from bytes to string, which should probably be fixed by adding a new variable instead.

Since we don't support Python 2, should we just remove this if statement?

Reply:
Yep, I think we can remove it.